rechunker/executors/python.py

```diff
@@ -0,0 +1,45 @@
+import itertools
+from functools import partial
+import math
+
+from typing import Any, Callable, Iterable
+
+from rechunker.types import CopySpec, StagedCopySpec, Executor
+
+
+Thunk = Callable[[], None]
```

Comment:
Those of us who are new to typing would appreciate a comment about what `Thunk` is supposed to be.

Reply:
"thunk" is just a name for a "function without any arguments", but the name doesn't provide much value here. See 
https://github.com/pangeo-data/rechunker/pull/33 for removing it.